app/server.js

```diff
@@ -53,7 +53,7 @@ app.use(compression());
 app.use(express.static(path.resolve(__dirname, '../dist'), { maxage }));
 app.use('/public', express.static(path.resolve(__dirname, '../public')));
 
-app.use(bodyParser.json({ limit: '1mb' }));
+app.use(/\/((?!remotepublic).)*/, bodyParser.json({ limit: '1mb' }));
```

Comment:
@konzz can you please remind us why this route required to be excluded from the Body Parser?

Reply:
So it's not "double-parsed" by the first and the second server.
So the first server receives the request in 'remotepublic' route, and does not parse it or does anything with it, just redirects it to the second server to the route 'public' where it gets correctly parsed.